src/20241105_AaveV3Ethereum_RemoveFraxFromIsolationModeOnAaveV3Mainnet/RemoveFraxFromIsolationModeOnAaveV3Mainnet.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Remove Frax from Isolation Mode on Aave v3 Mainnet"
+author: "Aave Chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-remove-frax-from-isolation-mode-on-aave-v3-mainnet/19337"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x9bc3f3d8e38d70f55887f2f2498e1b39f59467489158923488aceab73cd4f144"
+---
+
+## Simple Summary
+
+This is an ARFC to remove FRAX from isolation mode, and set LTV to 0%. This is a reboot with minor changes of a [previous ARFC](https://governance.aave.com/t/arfc-remove-frax-from-isolation-mode-and-onboard-sfrax-to-aave-v3-mainnet/18506) which passed Snapshot but has not yet been implemented.
+
+## Motivation
+
+FRAX and Aave DAO have found more synergies over the last months. The FRAX team has responded with major updates to security on sfrxETH in response to BGD Labs feedback. FRAX has also initated governance proposals to add GHO to Frax Lend. There are ongoing conversations to have a FRAX AMO included into Aave v3. sFRAX was [previously accepted](https://governance.aave.com/t/arfc-add-sfrax-on-ethereum-v3/16303) for onboarding in a previous [ARFC vote](https://snapshot.org/#/aave.eth/proposal/0xdba99e9c8da24424447d7c7b70eff93ad5b6055714b5f34cf9859c923fb3a38a) before the introduction of CAPO feeds.
+
+This proposal suggests removing FRAX from isolation mode to facilitate further AMO deployments.
+
+Furthermore, this proposal sets LTV to 0% as there is no use-case for FRAX to act as collateral.
+
+## Specification
+
+- FRAX will be removed from Isolation Mode on Aave v3 instances.
+- FRAX will receive same parameters as DAI, a Reserve Factor as other stables, and LTV of 0%.
+
+| Parameter                | Recommendation |
+| ------------------------ | -------------- |
+| Isolation Mode           | NO             |
+| Borrowable in Isolation  | NO             |
+| Stable Borrow            | NO             |
+| Enable Borrow            | NO             |
+| Enable Collateral        | YES            |
```

Comment:
Tricky to say it is enabled as collateral as LTV is 0

Reply:
I believe enable borrow & collat got mixed up, I left it as is while I double check with risk